docs/UserGuide.md

```diff
@@ -721,15 +770,13 @@ Important notes:
 > Location: Bugis;
 > Categories: #transport
 > ```
-> Input:
-> <img src="images/unicash/command-outputs/getTransaction/getInitial1.png" width="1000" />
-> Ouput:
-> <img src="images/unicash/command-outputs/getTransaction/getSuccessFinal1.png" width="1000" />
 
 ###### Example 2
 
-> **Case**: Retrieve a transaction with the currently not visible but correctly specified `<INDEX>`.
->
+> **Case**: Retrieve a transaction with the currently not visible but correctly specified
+> `INDEX`. The transaction is present in the current `Transactions List` configuration 
+> but the user has scrolled further away.
+> 
```

Comment:
Same comment as the DELETE comment above where this seems like it should be a callout above instead of an explicit case